<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrammarView</class>
 <widget class="QWidget" name="GrammarView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grammar Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hori2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout5">
       <item>
        <widget class="QLabel" name="lbStructures">
         <property name="text">
          <string>Sentences:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SelectionListView" name="lvStructures"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vertiLayout10">
       <item>
        <widget class="QLabel" name="lbExamples">
         <property name="text">
          <string>Examples with your vocabulary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwExamples"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="pbAdd">
         <property name="text">
          <string>Add sentence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDelete">
         <property name="text">
          <string>Delete sentence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbImportTexts">
         <property name="text">
          <string>Import</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbRename">
         <property name="text">
          <string>Rename Category</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbMerge">
         <property name="toolTip">
          <string>Using this procedure you can keep your grammar structures to a minimum and improve the readability of grammar and vocabulary.</string>
         </property>
         <property name="text">
          <string>Merge Categories</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectionListView</class>
   <extends>QListView</extends>
   <header location="global">simonuicomponents/selectionlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
